acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -6,11 +6,12 @@ module Puppet
   module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
-        :redhat        => /fedora|el|centos/,
         :debian        => /debian|ubuntu/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :redhat        => /fedora|el|centos/,
+        :sles          => /sles/,
         :solaris       => /solaris/,
-        :windows       => /windows/,
+        :windows       => /windows/
```

Comment:
I can't tell you how happy I am that you're running rubocop on this code :+1: 

That said, yeah, we have `Style/TrailingComma` off in the puppet rubocop config. (Although in fairness, we have a *lot* of cops off ...)